keyboards/handwired/pteron/config.h

```diff
@@ -0,0 +1,37 @@
+#ifndef CONFIG_H
+#define CONFIG_H
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0xFEED
+#define PRODUCT_ID      0x6060
+#define DEVICE_VER      0x0001
+// #define MANUFACTURER    qmkbuilder
```

Comment:
This still needs to be there.  
If you want "QMK Community" is a good option for this.